From 234a62418c56cea749e8b86618ea06cbf5cfc2c9 Mon Sep 17 00:00:00 2001
From: Leonardo Fagundes Luz Serrano
 <Leonardo.FagundesLuzSerrano@windriver.com>
Date: Tue, 25 Jan 2022 19:55:28 -0300
Subject: [PATCH] Fix error related to /src/.git_version

Error message:
"force parsing /<<PKGBUILDDIR>>/src/.git_version
for CEPH_GIT_VER and CEPH_GIT_NICE_VER
CMake Error at src/CMakeLists.txt:183 (file):
file STRINGS file "/<<PKGBUILDDIR>>/src/.git_version"
cannot be read."

Signed-off-by: Leonardo Fagundes Luz Serrano <Leonardo.FagundesLuzSerrano@windriver.com>
---
 src/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index be06fe5016..3c26d1d7f7 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -177,7 +177,7 @@ set(GCOV_PREFIX_STRIP 4)
 
 # the src/.git_version file may be written out by make-dist; otherwise
 # we pull the git version from .git
-option(ENABLE_GIT_VERSION "build Ceph with git version string" ON)
+option(ENABLE_GIT_VERSION "build Ceph with git version string" OFF)
 if(${ENABLE_GIT_VERSION})
    message(STATUS "force parsing ${CMAKE_CURRENT_SOURCE_DIR}/.git_version for CEPH_GIT_VER and CEPH_GIT_NICE_VER")
    file(STRINGS ${CMAKE_CURRENT_SOURCE_DIR}/.git_version CEPH_GIT_SHA_AND_TAG)
-- 
2.17.1

